platformerLib/GameEngine.cpp

```diff
@@ -46,7 +46,8 @@ void GameEngine::CheckGraySquareActorLocation(PhysicalObject* actor,
 
 	Distance new_x(actor->GetLocation().x.Value());
 	Distance new_y(actor->GetLocation().y.Value());
-	bool is_touched = false;
+   bool is_touched_horizontally = false;
```

Comment:
Yes, but we shouldn't leave both spaces and tabs in the source file (it looks ugly).
Please, make the same style in the whole file.